.github/workflows/build-sample-apps.yml

```diff
@@ -124,11 +125,10 @@ jobs:
         GOOGLE_CLOUD_MATCH_READONLY_SERVICE_ACCOUNT_B64: ${{ secrets.GOOGLE_CLOUD_MATCH_READONLY_SERVICE_ACCOUNT_B64 }}
         FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64: ${{ secrets.FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64 }}
 
-    - name: Setup to determine binary size of SDK
-      working-directory: Apps/${{ matrix.sample-app }}/build/    
-      run: |
-        brew install bloaty
-        mv *.xcarchive App.xcarchive # rename the file to a static value. Bloaty requires a hard-coded path to files to work. 
+    # xcodebuild creates builds that include a timestamp in the name. In order for bloaty to read the build, we need to rename it to a static name.
+    - name: Rename the same app build to a static name that bloaty can understand  
+      working-directory: Apps/${{ matrix.sample-app }}/build/
+      run: mv *.xcarchive App.xcarchive
```

Comment:
can there be multiple `.xcarchive` generated? or is it only one with timestamp in the name?

Reply:
A new `.xcarchive` directory gets generated each time we call `fastlane ios build`. Which, we only run once in this job. 

If you're concerned about what would happen if there are multiple `.xcarchive` directories in the `build/` directory, from my testing, the `mv` command fails if multiple `.xcarchive` directories are found.